### PR TITLE
feat: automatically join related projects when visiting place

### PR DESCRIPTION
**Pull request by hhartz**

### CODE REVIEW

File: ios/SjekkUt.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -255,6 +256,7 @@
 		6CE0763B1D4763B8000ACAAA /* Project.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; name = Project.m; path = model/Project.m; sourceTree = "<group>"; };
 		6CE2966D1D90005800A73967 /* DntCardView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = DntCardView.swift; path = views/DntCardView.swift; sourceTree = "<group>"; };
 		6CE526901D507BA700D3419B /* DntExtensions.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = DntExtensions.swift; sourceTree = "<group>"; };
+		6CE6F0AD1D9E7C2D00D3F48A /* tryvannsstua.gpx */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.xml; name = tryvannsstua.gpx; path = "position-gpx/tryvannsstua.gpx"; sourceTree = SOURCE_ROOT; };
```
Comment (snowpong) [inline ios/SjekkUt.xcodeproj/project.pbxproj]: I assume changes in this file we ignore?

File: ios/SjekkUt/network/SjekkUtApi.swift [hunk 2]
```diff
@@ -242,6 +241,11 @@ class SjekkUtApi: DntManager {
                             NSNotificationCenter.defaultCenter().postNotificationName(SjekkUtCheckedInNotification, object:checkin);
                             DntAnalytics.track("Server registrerte besøk", ["placeid":"\(aPlace.identifier!)"])
                         }
+                        for project:Project in aPlace.projects ?? [] {
```
Comment (snowpong) [inline ios/SjekkUt/network/SjekkUtApi.swift]: Let's discuss the question marks

File: ios/position-gpx/tryvannsstua.gpx [hunk 3]
```diff
@@ -0,0 +1 @@
...
xmlns="http://www.topografix.com/GPX/1/1"
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd"
version="1.1" 
creator="gpx-poi.com">
   <wpt lat="59.997842" lon="10.668684">
      <time>2015-02-09T12:48:25Z</time>
      <name>Tryvannsstua</name>
   </wpt>
</gpx>
```
Comment (snowpong) [inline ios/position-gpx/tryvannsstua.gpx]: Why do we add  this file at all? Doesn't seem related to change

File: ios/SjekkUt/network/SjekkUtApi.swift (hunk 2 above)
Comment (hhartz) [inline ios/SjekkUt/network/SjekkUtApi.swift]: this is the "nil coalescing" operator, basically a default value in case the Optional is `nil`

File: ios/SjekkUt.xcodeproj/project.pbxproj (hunk 1 above)
Comment (hhartz) [inline ios/SjekkUt.xcodeproj/project.pbxproj]: this lets the simulator set location to one of the places in a list, and check in to it

File: ios/position-gpx/tryvannsstua.gpx (hunk 3 above)
Comment (hhartz) [inline ios/position-gpx/tryvannsstua.gpx]: this lets the simulator set location to one of the places in a list, and check in to it
